src/components/Datatable/Datatable.jsx

```diff
@@ -100,12 +100,13 @@ const DataTable = ({
                     },
                   }
                 }
-                className="ds-u-padding--2 ds-u-border-y--2 ds-u-border--dark  ds-u-font-weight--bold dc-c-table-header-cell"
+                className="ds-u-border-y--2 ds-u-border--dark  ds-u-font-weight--bold dc-c-table-header-cell"
```

Comment:
This may need to be changed to just `ds-u-padding-y-2` if the left/right padding isn't needed.